test/integration/Distributions/runtests.jl

```diff
@@ -0,0 +1,420 @@
+        TestCase(TriangularDist(0.1, 1.4, 0.45), 0.12),
+        TestCase(TriangularDist(0.0, 1.5, 0.5), 0.2),
+        TestCase(Triweight(1.0, 1.0), 1.0),
+        TestCase(Triweight(1.1, 2.1), 1.0),
+        TestCase(Triweight(1.9, 10.0), -0.1),
+        TestCase(Uniform(0.0, 1.0), 0.2),
+        TestCase(Uniform(-0.1, 1.1), 1.0),
+        TestCase(Uniform(99.5, 100.5), 100.0),
+        TestCase(VonMises(0.5), 0.1),
+        TestCase(VonMises(0.3), -0.1),
+        TestCase(VonMises(0.2), -0.5),
+        TestCase(Weibull(0.5, 1.0), 0.45),
+        TestCase(Weibull(0.3, 1.1), 0.66),
+        TestCase(Weibull(0.75, 1.3), 0.99),
+
+        #
+        # Multivariate
+        #
+
+        TestCase(MvNormal(1, 1.5), [-0.3]),
+        TestCase(MvNormal(2, 0.5), [0.2, -0.3]),
+        TestCase(MvNormal([1.0]), [-0.1]),
+        TestCase(MvNormal([1.0, 0.9]), [-0.1, -0.7]),
+        TestCase(MvNormal([0.0], 0.9), [0.1]),
+        TestCase(MvNormal([0.0, 0.1], 0.9), [0.1, -0.05]),
+        TestCase(MvNormal(Diagonal([0.1])), [0.1]),
+        TestCase(MvNormal(Diagonal([0.1, 0.2])), [0.1, 0.15]),
+        TestCase(MvNormal([0.1, -0.3], Diagonal(Fill(0.9, 2))), [0.1, -0.1]),
+        TestCase(MvNormal([0.1, -0.1], 0.4I), [-0.1, 0.15]),
+        TestCase(MvNormal([0.2, 0.3], Hermitian(Diagonal([0.5, 0.4]))), [-0.1, 0.05]),
+        TestCase(MvNormal([0.2, 0.3], Symmetric(Diagonal([0.5, 0.4]))), [-0.1, 0.05]),
+        TestCase(MvNormal([0.2, 0.3], Diagonal([0.5, 0.4])), [-0.1, 0.05]),
+        # TODO Broken tests, see https://github.com/EnzymeAD/Enzyme.jl/issues/1991
+        TestCase(MvNormal([-0.15], _pdmat([1.1]')), [-0.05]; broken=Forward),
+        TestCase(
+            MvNormal([0.2, -0.15], _pdmat([1.0 0.9; 0.7 1.1])), [0.05, -0.05];
+            broken=Forward
+        ),
+        TestCase(MvNormal([0.2, -0.3], [0.5, 0.6]), [0.4, -0.3]),
+        TestCase(MvNormalCanon([0.1, -0.1], _pdmat([0.5 0.4; 0.45 1.0])), [0.2, -0.25]),
+        # TODO Broken tests, see https://github.com/EnzymeAD/Enzyme.jl/issues/1991
+        TestCase(
+            MvLogNormal(MvNormal([0.2, -0.1], _pdmat([1.0 0.9; 0.7 1.1]))), [0.5, 0.1];
+            broken=Forward
+        ),
+        TestCase(product_distribution([Normal()]), [0.3]),
+        TestCase(
+            product_distribution([Normal(), Uniform()]), [-0.4, 0.3];
+            runtime_activity=Both
+        ),
+
+        #
+        # Matrix-variate
+        #
+
+        TestCase(
+            MatrixNormal(
+                randn(rng, 2, 3), _pdmat(randn(rng, 2, 2)), _pdmat(randn(rng, 3, 3))
+            ),
+            randn(rng, 2, 3),
+        ),
+        # TODO In addition to Forward mode being broken the "normal way" for the below one,
+        # Reverse mode also flat out crashes Julia. Hence the commenting out.
+        # See https://github.com/EnzymeAD/Enzyme.jl/issues/2012.
+        # TODO Broken tests, see https://github.com/EnzymeAD/Enzyme.jl/issues/1821
+        # TestCase(
```

Comment:
This should be commented in as it presumably no longer crashes

Reply:
Done. Also marked another test as not broken any more.